Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIEventCocoa.mm

```diff
@@ -43,13 +43,13 @@
 
 void WebExtensionContext::addListener(WebPageProxyIdentifier identifier, WebExtensionEventListenerType type, WebExtensionContentWorldType contentWorldType)
 {
-    auto page = WebProcessProxy::webPage(identifier);
+    RefPtr page = WebProcessProxy::webPage(identifier);
```

Comment:
This is not strictly necessary since `WebProcessProxy::webPage()` returns a `RefPtr<WebProcessProxy>`, but I'm fine making this more explicit.

<https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/WebProcessProxy.h#L204>

Reply:
@cdumez has asked to be explicit like this, and not use `auto` in these cases.